src/qutip_qip/algorithms/qft.py

```diff
@@ -133,6 +133,56 @@ def qft_gate_sequence(N=1, swapping=True, to_cnot=False):
                 qc.add_gate("SWAP", targets=[N - i - 1, i])
     return qc
 
+def inverse_qft_gate_sequence(N=1, swapping=True, to_cnot=False):
```

Comment:
Nice, is it possible to merge this into the `qft_gate_sequence`? They share the same input arguments. We could define `qft_gate_sequence(N=1, swapping=True, to_cnot=False, inverse=False)`, which returns the traditional qft if inverse=False and the inversed one if True.

The code also looks very similar, maybe we can even merge most of the code? I feel like there are only a few minus sign which can be defined according to the value of `inverse`. Please correct me if I and wrong.

Reply:
I tried doing this, the solution doesn't look elegant

Reply:
Ah that makes sense!